### ISSUE TITLE
Bootloader entered in to infinite loop.

### ISSUE THREAD

**Issue by jkandasa** [mysensors/MySensorsBootloaderRF24#7]
When we do not have any node/gateway around the reachable area, bootloader gets stuck on searching something. I guess it's waiting for parent node id?

In this situation bootloader never enter into the application/user program.

I use `1.3.0-beta.3` bootloader.

**Comment by tekka007** [comment]
@jkandasa No, it should timeout and proceed to startup check, https://github.com/mysensors/MySensorsBootloaderRF24/blob/development/MySensorsBootloader.h#L158-L159

What are the observations supporting your conclusion?

**Comment by jkandasa** [comment]
@tekka007 I did not check the code. When I turn on only one node(no other nodes on surrounding) with this bootloader, it is not going to the application. It is on with bootloader. When I turn ON any node. it goes immediately.

This behaviour makes me think that bootloader has some infinitive loop.
